neuralmonkey/model/model_part.py

```diff
@@ -51,7 +51,7 @@ def feed_dict(self, dataset: Dataset, train: bool) -> FeedDict:
     def _init_saver(self) -> None:
         if not self._saver:
             parts_variables = tf.get_collection(
-                tf.GraphKeys.GLOBAL_VARIABLES, scope=self._variable_scope.name)
+                tf.GraphKeys.TRAINABLE_VARIABLES, scope=self._variable_scope.name)
```

Comment:
kazdopadne GLOBAL_VARIABLES je take nepouzitelne protoze to uklada zase az moc a nejde to nacist samostatne bez napriklad optimizeru, coz predpokladam nechceme. Takze co s tim? ja se nevyznam ve vsech tech promenych, jen jsem testoval tuhle a ta ulozila vse pro danou konvolucni sit.